scripts/smoke-test-droid.ps1

```diff
@@ -0,0 +1,112 @@
+# GITHUB_WORKSPACE is the root folder where the project is stored.
+Write-Output "#################################################"
+Write-Output "#   ANDROID                                     #"
+Write-Output "#            VALIDATOR                          #"
+Write-Output "#                       SCRIPT                  #"
+Write-Output "#################################################"
+
+Set-Variable -Name "ApkPath" -Value "samples/artifacts/builds/Android"
+Set-Variable -Name "ApkFileName" -Value "IL2CPP_Player.apk"
+Set-Variable -Name "ActivityName" -Value "io.sentry.samples.unityofbugs"
+Set-Variable -Name "TestActivityName" -Value "io.sentry.samples.unityofbugs/com.unity3d.player.UnityPlayerActivity"
+
+# Filter device List
+$RawAdbDeviceList = adb devices
```

Comment:
I'm not familiar with adb devices, is there a way to install additional ones?

Reply:
`adb devices` gives the list of devices. if you have one device connected, it'll display only one item.
In the case of multiple devices, it'll display all active devices.
You need that info in case of having multiple active simulators because if you have more than one device connected, you need to specify which device you wish to execute the command.